### PR TITLE
Add new channel flag (OBFUSCATED)

### PR DESCRIPTION
**Pull request by BoryaGames**
Discord has added a new channel flag in [this commit](https://github.com/Discord-Datamining/Discord-Datamining/commit/4338e84cf234c6d4ad008bbe1ef7dc2dec877d48#commitcomment-147964318).
image

### CODE REVIEW

File: pages/resources/channel.mdx [hunk 1]
```diff
@@ -109,6 +109,7 @@ Type `10`, `11` and `12` are only available in API v9 and above.
 | 1 << 14     | BROADCASTING                               | This group DM is used for broadcasting a live stream                                                                                                            |
 | 1 << 15     | HIDE_MEDIA_DOWNLOAD_OPTIONS                | This media channel has the embedded download options hidden for media attachments                                                                               |
 | 1 << 16     | JOIN_REQUEST_INTERVIEW_CHANNEL             | This group DM is used for [guild join request interviews](https://support.discord.com/hc/en-us/articles/23187611406999-Guilds-FAQ#h_01HXW2MCD0BT70FB9SRV1YRPBV) |
+| 1 << 17     | OBFUSCATED                                 | This channel's name is obfuscated because user doesn't have access to it                                                                                        |
```
Comment (splatterxl) [inline pages/resources/channel.mdx:112]:
```suggestion
| 1 << 17     | OBFUSCATED                                 | This channel is unavailable for the current user                                                                                        |
```

Can you add a footnote saying something like ``The channel's name is always returned as `___hidden___`.``

**Review by splatterxl — CHANGES_REQUESTED** [review]

File: pages/resources/channel.mdx [hunk 2]
```diff
@@ -109,9 +109,12 @@ Type `10`, `11` and `12` are only available in API v9 and above.
 | 1 << 14     | BROADCASTING                               | This group DM is used for broadcasting a live stream                                                                                                            |
 | 1 << 15     | HIDE_MEDIA_DOWNLOAD_OPTIONS                | This media channel has the embedded download options hidden for media attachments                                                                               |
 | 1 << 16     | JOIN_REQUEST_INTERVIEW_CHANNEL             | This group DM is used for [guild join request interviews](https://support.discord.com/hc/en-us/articles/23187611406999-Guilds-FAQ#h_01HXW2MCD0BT70FB9SRV1YRPBV) |
+| 1 << 17     | OBFUSCATED ^2^                             | This channel is unavailable for the current user                                                                                                                |
```
Comment (splatterxl) [inline pages/resources/channel.mdx:112]:
```suggestion
| 1 << 17     | OBFUSCATED ^2^                             | This channel is unavailable for the current user, and has its details obfuscated                                                                                                                |
```

File: pages/resources/channel.mdx [hunk 3]
```diff
@@ -109,9 +109,12 @@ Type `10`, `11` and `12` are only available in API v9 and above.
...
 
 ^1^ Media channels are now represented by the [`GUILD_MEDIA` channel type](#channel-type).
 
+^2^ The channel's name is always returned as `___hidden___`.
```
Comment (splatterxl) [inline pages/resources/channel.mdx:116]:
```suggestion
^2^ The channel's name and topic are always returned as `___hidden___`.
```

**Review by splatterxl — CHANGES_REQUESTED** [review]

**Review by splatterxl — APPROVED** [review]

**Comment by BoryaGames** [comment]
Resolved merge conflict

**Review by Dziurwa14 — APPROVED** [review]

**Review by mantikafasi — APPROVED** [review]

**Comment by voidfill** [comment]
LGTM

**Review by splatterxl — APPROVED** [review]

**Comment by splatterxl** [comment]
Thank you!

**Pull request merged**